### PR TITLE
[quant][fix][fx][graphmode] Fix qconfig setting for fused modules

### PR DESCRIPTION
**Pull request by jerryzh168**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #71254

Summary:
when we configure linear and relu with the same qconfig, we currently have utility functions to also
generate a qconfig for the fused linear relu module, but this code is not called in correct order before
which resulted in unexpected behaviors. This PR fixes the issue. Please see test case for more details.
(Test case is from Supriya)

Test Plan:
python test/test_quantization.py TestQuantizeFx.test_fused_module_qat_swap

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D33558321](https://our.internmc.facebook.com/intern/diff/D33558321)

### CODE REVIEW

**Comment by jerryzh168** [comment]
@jerryzh168 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33558321).

File: test/quantization/fx/test_quantize_fx.py [hunk 1]
```diff
@@ -496,6 +496,53 @@ def forward(self, x, y):
...
+    def test_fused_module_qat_swap(self):
+        class Linear(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.w = torch.ones(5, 5)
+                self.b = torch.zeros(5)
+
+            def forward(self, x):
+                return torch.nn.functional.linear(x, self.w, self.b)
+
```
Comment (supriyar) [inline test/quantization/fx/test_quantize_fx.py:508]: I don't think this is used in the model

**Review by supriyar — APPROVED** [review]

File: test/quantization/fx/test_quantize_fx.py (hunk 1 above)
Comment (jerryzh168) [reply test/quantization/fx/test_quantize_fx.py:508]: oh OK, will remove

**Comment by jerryzh168** [comment]
@jerryzh168 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33558321).

**Comment by jerryzh168** [comment]
@jerryzh168 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33558321).

**Comment by jerryzh168** [comment]
@jerryzh168 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33558321).

**Pull request closed without merging**